app/assets/stylesheets/new_board/_history.scss

```diff
@@ -1,38 +1,74 @@
-.history-activity {
-  background-color: $lightgrey-11;
-  border-bottom: 1px solid $lightgrey-13;
-  border-top: 1px solid $white;
-  min-height: 37px;
-  padding: 7px 8px;
-  font-size: 11px;
+.History {
+  position: relative;
 
-  .header {
-    display: flex;
-    justify-content: space-between;
-
-    .title {
-      font-size: 14px;
-    }
-
-    .date {
-      font-size: 10px;
-      font-weight: bold;
-    }
+  &:before {
+    content: "";
+    width: 3px;
+    background: $orange;
+    position: absolute;
+    left: 8px;
+    top: 1%;
+    border-radius: 60%;
+    transition: ease-in-out .2s;
+    z-index: 1;
+    animation-duration: 3s;
+    animation-fill-mode: both;
+    animation-name: heightAnimation;
   }
 
-  .info {
-    display: flex;
-    flex-direction: column;
-    margin: 0.5em 2em 0.5em 1em;
+  .history-activity {
+    background-color: $lightgrey-11;
+    border-bottom: 1px solid $lightgrey-13;
+    border-top: 1px solid $white;
+    min-height: 37px;
+    padding: 7px 8px;
+    font-size: 11px;
+    padding-left: 20px;
+    position: relative;
 
-    .action {
-      font-weight: bold;
+    &:before {
+      content: "";
+      width: 10px;
+      height: 10px;
+      background: $black;
+      z-index: 2;
+      position: absolute;
+      border-radius: 100%;
+      top: 9px;
+      left: 4px;
     }
-
-    .changes {
+  
+    .header {
       display: flex;
-      font-style: italic;
-
+      justify-content: space-between;
+  
+      .title {
```

Comment:
Can we use BEM selectors here? I know you weren't the one who created those but we can refactor it now.

Reply:
Sure!